ntbs-integration-tests/Helpers/TestDocumentExtensions.cs

```diff
@@ -6,6 +6,6 @@ namespace ntbs_integration_tests.Helpers
     public static class HtmlDocumentExtensions
     {
         public static string GetError(this IHtmlDocument document, string input) => 
-            document.QuerySelector($"span[id='{input}-error']").TextContent;
+            document?.QuerySelector($"span[id='{input}-error']")?.TextContent;

```

Comment:
@ChristianMcc @jasiekmiko 
Are we okay with this?
Think a test fail with "expected string" is not equal to "null" or whatever it comes up with, would be more useful than a null exception

Reply:
Sounds good, probably a bit easier to debug. Trying to think if we would ever want to test the case that the TextContent is empty/null on an element that does exist (and therefore we could get a false positive with this method), but I think if the element exists and TextContent is empty we would be testing for an empty string not null so shouldn't be an issue? Let me know if this hasn't made any sense!

Reply:
Yeah, if the span does exist it always has "Error:" in it anyway (regardless of whether an error occurs). regardless of whether an actual error occurs so the tester would have to bear that in mind.